ci-operator/step-registry/single-node/conf/e2e/graceful-shutdown/single-node-conf-e2e-graceful-shutdown-commands.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -o errexit
```

Comment:
@rphillips why do we a dedicated job? Doesn't it make sense that we always use it in SNO?

Reply:
we want to test it for SNO for the time being. It'll be super simple to migrate it over.

Reply:
sorry, I think I was not clear. Why we cannot add this config to the default SNO (e2e-aws-single-node) job. Why to add another one

Reply:
We wanted to create a new CI target to make sure everything is working correctly.